lib/ask/runtime/survey_broker.ex

```diff
@@ -372,7 +371,13 @@ defmodule Ask.Runtime.SurveyBroker do
   end
 
   defp start_some(survey, count) do
-    count = Enum.min([batch_limit_per_minute(survey.project), count])
+    count_limit = batch_limit_per_minute(survey.project)
+    if count > count_limit do
+      Logger.info("Survey #{survey.id}. Starting up to #{count_limit} respondents due to batch limit (wanted to start #{count}).")
+      count = count_limit
```

Comment:
the compiler is complaining about this. I think that we should do something like this instead 
```elixir
count = if count > count_limit do
...
end
```